linux.version

```diff
@@ -1,11 +1,11 @@
 {
   "linux": {
-    "version": "2.31.11",
+    "version": "2.31.12",
     "latest": "true",
     "build": "1",
     "fluent-bit": "1.9.10",
     "kinesis-plugin": "v1.10.1",
     "firehose-plugin": "v1.7.1",
-    "cloudwatch-plugin": "v1.9.2"
+    "cloudwatch-plugin": "v1.9.4"
```

Comment:
this is for the bug fix from our recent ticket, see changelog entry in this same PR

Reply:
Skipping from 1.9.2 to 1.9.3 means that our previous release failed to upgrade to 1.9.3 like it was supposed to. I checked our release note tho and this does not really impact customers because 1.9.3 was just a go version upgrade: https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit/releases/tag/v1.9.3